packages/gui/src/hooks/useNFTMetadata.ts

```diff
@@ -1,23 +1,43 @@
 import { useEffect, useState, useCallback } from 'react';
 import type NFTInfo from '@chia/api';
-import useVerifyURIHash from './useVerifyURIHash';
-import getRemoteFileContent from '../util/getRemoteFileContent';
+import getRemoteFileContent, { FileType } from '../util/getRemoteFileContent';
```

Comment:
`FileType` isn't used

```suggestion
import getRemoteFileContent from '../util/getRemoteFileContent';
```